issuer-front-end/.yarnrc.yml

```diff
@@ -0,0 +1 @@
+nodeLinker: node-modules
```

Comment:
The recommended way to specify which version of yarn used in the project, is to have a line in this file pointing to the yarn release, which should be included in the repository. Normally, you would achieve this by running

```bash
yarn set version 3.2.0
```

to match the version specified in `package.json`

Another way to do this is, when initializing the project:

```bash
yarn init -2
```